assets/css/layouts/home.css

```diff
@@ -95,7 +95,7 @@
 	transition: all 0.2s;
 }
 .playground-header-figure-wrapper img {
-	transition: all 0.3s;
+	transition: all 0.1s;
 }
 .playground-header-figure-wrapper.loading img {
 	opacity: 0;
```

Comment:
Something is wrong about the transition. If I simulate a slow connection (200 ms extra wait time for the `fetch` request), I get flickering like this in Safari:

https://github.com/getkirby/getkirby.com/assets/1595007/82687b9d-70b9-4a29-afc4-ef0ce7ab8174

Other browsers don't seem to have that issue, but in Safari it happens all the time. Do you also see that?

Reply:
I have it in Firefox as well that sometime the fade-in animation is shown, but sometimes not at all.

Reply:
Splitting the image loading and the fade-in to two separate ticks seems to do the trick.